guide/special.md

```diff
@@ -0,0 +1,115 @@
+# Special Characters
+
+In Fluent, text is the most important part of the file. As such, it doesn't
+have any special syntax: you can just write it without any delimiters (e.g.
+quotes). It also doesn't recognize any special escape sequences. Regular text
```

Comment:
Regular text: should we explicitly mention unicode somewhere?

Reply:
There's a whole paragraph about Unicode at the bottom of this file, but I think you're right: it's worth adding a note about it here as well.